README.md

```diff
@@ -2,11 +2,29 @@
 
 This project contains the MiSTer core for Cave's first-generation 68k arcade hardware. 
 
-It currently supports the following games:
-
-* DoDonPachi
-* ~~DonPachi~~
-* ~~Dangun Feveron~~
+![23](https://user-images.githubusercontent.com/32810066/126911397-e54b786a-39ff-4200-8fdb-8750d1b40976.png)
+
+## Supported Games
+
+| Title | Status | Released |
+|------|---------|----------|
+[**Do Donpachi**](https://en.wikipedia.org/wiki/DoDonPachi) | Implemented | Yes |
+[**ESP Ra. De.**](https://en.wikipedia.org/wiki/ESP_Ra.De.) | Implemented | Yes |
+[**Dangun Feveron**](https://en.wikipedia.org/wiki/Dangun_Feveron) | **W.I.P** | No |
+[**Air Gallet**](https://en.wikipedia.org/wiki/Air_Gallet) | N/A | No |
+[**Donpachi**](https://en.wikipedia.org/wiki/DonPachi) |  N/A | No |
+[**Gaia Crusaiders**](http://adb.arcadeitalia.net/dettaglio_mame.php?game_name=gaia&arcade_only=0&autosearch=1) | N/A | No |
+[**Gunwange**](https://en.wikipedia.org/wiki/Guwange) | N/A | No |
```

Comment:
pretty much sure there was a beta of this?

Reply:
If those are in beta, then I can update the status for those games. Maybe @nullobject can confirm